package.json

```diff
@@ -59,6 +58,8 @@
     "ember-resolver": "^4.3.0",
     "ember-sinon": "^1.0.0",
     "ember-source": "~2.16.0",
+    "fastboot-transform": "^0.1.2",
```

Comment:
@kratiahuja shouldn't `fastboot-transform` be listed in `dependencies`? If I understand correctly it needs to be available for the application at build time.

Reply:
Yes,confirm

Reply:
Thank you @rwjblue for confirmation, I’ll make PR to fix this